README.md

```diff
@@ -218,15 +220,17 @@ For **all other services** it is clearly visible that they are working with a no
 requests rate is much higher than the count of active threads would allow it to be. This seems to work a bit better for
 micronaut and ktor than for spring reactive and node. Vertx is the clear winner of this test. At 700 req/sec it still
 responds nearly as fast as the mocked service. With higher request counts the response times of Vertx get noticeably
-higher, but it stays responsive up until impressing 1000 req/sec.
+higher, but it stays responsive up to impressing 1000 req/sec.
+
+<mark>I wonder if a normalized comparison here would have been more useful, i.e. showing req/thread/sec.</mark>
```

Comment:
But wouldn't this normalization make the differences even bigger, since the service using the most threads is also able to serve the least requests?

I think a normalization over the used resources would express the most, but this is quite hard to achieve for which reason I added the third test.